src/psyclone/tests/psyir/nodes/psy_data_node_test.py

```diff
@@ -57,86 +58,27 @@ def clear_psydata_namespace():
 
 
 # -----------------------------------------------------------------------------
-def test_psy_data_basic(capsys):
-    # pylint: disable=too-many-locals
-    '''Check basic functionality: node names, schedule view.
-    '''
-    from psyclone.psyir.nodes.node import colored, SCHEDULE_COLOUR_MAP
-    _, invoke = get_invoke("test11_different_iterates_over_one_invoke.f90",
-                           "gocean1.0", idx=0)
-    schedule = invoke.schedule
-
-    data_trans = PSyDataTrans()
-    assert "Insert a PSyData node" in str(data_trans)
-    assert data_trans.name == "PSyDataTrans"
-    data_trans.apply(schedule)
-
-    assert isinstance(invoke.schedule[0], PSyDataNode)
-
-    schedule.view()
-    out, _ = capsys.readouterr()
-
-    gsched = colored("GOInvokeSchedule", SCHEDULE_COLOUR_MAP["Schedule"])
-    sched = colored("Schedule", SCHEDULE_COLOUR_MAP["Schedule"])
-    loop = Loop().coloured_name(True)
-    data = invoke.schedule[0].coloured_name(True)
-
-    # Do one test based on schedule view, to make sure colouring
-    # and indentation is correct
-    expected = (
-        gsched + "[invoke='invoke_0', Constant loop bounds=True]\n"
-        "    0: " + data + "[]\n"
-        "        " + sched + "[]\n"
-        "            0: " + loop + "[type='outer', field_space='go_cv', "
-        "it_space='go_internal_pts']\n")
-    assert expected in out
-
-    # Insert a DataTrans call between outer and inner loop.
-    # This tests that we find the subroutine node even
-    # if it is not the immediate parent.
-    new_sched, _ = data_trans.apply(invoke.schedule[0].psy_data_body[0]
-                                    .loop_body[0])
-
-    new_sched_str = str(new_sched)
-    correct = ("""GOInvokeSchedule[invoke='invoke_0', \
-Constant loop bounds=True]:
-PSyDataStart[var=psy_data]
-GOLoop[id:'', variable:'j', loop_type:'outer']
-Literal[value:'2', DataType.INTEGER]
-Literal[value:'jstop-1', DataType.INTEGER]
-Literal[value:'1', DataType.INTEGER]
-Schedule:
-PSyDataStart[var=psy_data_1]
-GOLoop[id:'', variable:'i', loop_type:'inner']
-Literal[value:'2', DataType.INTEGER]
-Literal[value:'istop', DataType.INTEGER]
-Literal[value:'1', DataType.INTEGER]
-Schedule:
-kern call: compute_cv_code
-End Schedule
-End GOLoop
-PSyDataEnd[var=psy_data_1]
-End Schedule
-End GOLoop
-GOLoop[id:'', variable:'j', loop_type:'outer']
-Literal[value:'1', DataType.INTEGER]
-Literal[value:'jstop+1', DataType.INTEGER]
-Literal[value:'1', DataType.INTEGER]
-Schedule:
-GOLoop[id:'', variable:'i', loop_type:'inner']
-Literal[value:'1', DataType.INTEGER]
-Literal[value:'istop+1', DataType.INTEGER]
-Literal[value:'1', DataType.INTEGER]
-Schedule:
-kern call: bc_ssh_code
-End Schedule
-End GOLoop
-End Schedule
-End GOLoop
-PSyDataEnd[var=psy_data]
-End Schedule""")
-
-    assert correct in new_sched_str
+def test_basics(monkeypatch):
```

Comment:
Although we have `__init__` files in our test directories (so they have package names to distinquish them), that won't help within e.g. `tests.psyir.nodes` so please could you put back the "psy_data" part of the name.

Reply:
OK, I've added the names back in (but I thought only the file names needed to be unique, not the actual test function names??)

Reply:
Thanks. As you'll have gathered, I got quite confused about naming. What I do know is that if we have unique names then we can just do `pytest -k <unique-name>` to run a given test which is handy.

Reply:
Ah, I only use this together with a test name :) Saves a bit of time collecting all the test cases :) Anyway, more descriptive names are fine with me.